### ISSUE TITLE
Server Error

### ISSUE THREAD

**Issue by ellodandra** [tachiyomiorg/tachiyomi-extensions#11978]
### Source information

Komikcast 1.2.23

### Steps to reproduce

1. Open Komikcast
2. Open any manga

### Expected behavior

Can read manga normally

### Actual behavior

Can't read manga normally cuz server 1 in website Komikcast got an error. Please move to server 2.
![Screenshot_2022-05-25-23-38-34-42](https://user-images.githubusercontent.com/81138524/170317102-43302984-8105-4989-8525-f288547583b7.png)
![Screenshot_2022-05-25-23-51-18-25](https://user-images.githubusercontent.com/81138524/170318181-1ff0a755-23e3-48bb-81f5-e11226f39d2a.png)
![Screenshot_2022-05-25-23-50-50-20](https://user-images.githubusercontent.com/81138524/170318189-5ff7baba-4617-4361-b5eb-69c6a7b19783.png)

### Tachiyomi version

0.13.4

### Android version

Android 7.1.1

### Other details

Please fix it as soon as posible.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.4](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by TheKingTermux** [comment]
![Screenshot_2022-06-09-12-40-45-20_afd66b7916d19a60815f56273408b73e](https://user-images.githubusercontent.com/50316075/172772762-98d50f40-b177-47b3-94b1-df1350b00c7f.jpg)
![Screenshot_2022-06-09-12-41-02-88_afd66b7916d19a60815f56273408b73e](https://user-images.githubusercontent.com/50316075/172772766-9d07cbb8-55bc-4e2d-80ec-1c0f0661019a.jpg)
![Screenshot_2022-06-09-12-41-17-65_afd66b7916d19a60815f56273408b73e](https://user-images.githubusercontent.com/50316075/172772773-5554cefa-40a3-4da2-8c8e-242fa8eba67b.jpg)

Punya saya aman gan

**Comment by vetleledaal** [comment]
After #12134 was merged this issue has been fixed and can be closed.
